container.go

```diff
@@ -482,6 +494,17 @@ func (c *Container) fetchState(cmd string) (State, error) {
 	return state, nil
 }
 
+func (c *Container) getSystemMountInfo() {
+	// check if /dev needs to be bind mounted from host /dev
+	for _, m := range c.mounts {
+		if m.Source == "/dev" && m.Destination == "/dev" && m.Type == "bind" {
+			c.systemMountsInfo.BindMountDev = true
```

Comment:
Don't you think we should maybe remove the mount from the list after we identified it as a special case ?

Reply:
Our hyperstart agent anyways ignores /dev in any case, if /dev is passed as a bind mount or not. I think we should leave it to the agent and not do any removal here.

Reply:
Yes I agree, let's not remove too much info.